templates/csharp/empty/teamsapp.yml.tpl

```diff
@@ -0,0 +1,45 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.1.0/yaml.schema.json
```

Comment:
Please consider updating yaml schema version to v1.5

Reply:
All right. Will update it next PR.